client/my-sites/email/titan-add-mailboxes/titan-unused-mailbox-notice.jsx

```diff
@@ -44,16 +39,13 @@ const TitanUnusedMailboxesNotice = ( {
 
 	return (
 		<Notice icon="notice" showDismiss={ false } status="is-warning" text={ text }>
-			<NoticeAction external={ linkIsExternal } onClick={ onFinishSetupClick }>
-				{ translate( 'Finish Setup' ) }
-			</NoticeAction>
+			<NoticeAction onClick={ onFinishSetupClick }>{ translate( 'Finish Setup' ) }</NoticeAction>
```

Comment:
Should this CTA also be `Set up mailbox`?
```suggestion
			<NoticeAction onClick={ onFinishSetupClick }>{ translate( 'Set up mailbox' ) }</NoticeAction>
```